src/objects/api/filters.py

```diff
@@ -1,15 +1,67 @@
+from django.utils.translation import ugettext_lazy as _
+
 from django_filters import filters
 from vng_api_common.filtersets import FilterSet
 from vng_api_common.utils import get_help_text
 
 from objects.core.models import Object
 
+from .constants import Operators
+from .utils import display_choice_values_for_help_text, is_number
+from .validators import validate_data_attrs
+
 
 class ObjectFilterSet(FilterSet):
     type = filters.CharFilter(
         field_name="object_type", help_text=get_help_text("core.Object", "object_type")
     )
+    data_attrs = filters.CharFilter(
+        method="filter_data_attrs",
+        validators=[validate_data_attrs],
+        help_text=_(
+            """Only include objects that have attributes with certain values.
+Data filtering expressions are comma-separated and are structured as follows:
+A valid parameter value has the form `key__operator__value`.
+`key` is the attribute name, `operator` is the comparison operator to be used and `value` is the attribute value.
+Note: Values can be string or numeric. Dates are not supported.
+
+Valid operator values are:
+%(operator_choices)s
+`value` may not contain double underscore or comma characters.
+`key` may not contain comma characters and includes double underscore only if it indicates nested attributes.
+
+Example: in order to display only objects with `height` equal to 100, query `data_attrs=height__exact__100`
+should be used. If `height` is nested inside `dimensions` attribute, query should look like
+`data_attrs=dimensions__height__exact__100`
```

Comment:
Is it possible to provide the query param `data_attrs` multiple times? And if yes, what is the effect of this?

Reply:
In this case only the last query param is applied. I suppose it's a default behaviour for django filters, for example, if you provide `type` query params multiple times, only the last param would be applied